api/apps/api/src/modules/projects/blm/project-blm.module.ts

```diff
@@ -6,9 +6,17 @@ import { BlmModule } from '@marxan-api/modules/blm';
 import { SetProjectBlmHandler } from './set-project-blm-handler';
 import { ProjectBlmSaga } from './project-blm.saga';
 import { ChangeBlmRangeHandler } from './change-blm-range.handler';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Project } from '@marxan-api/modules/projects/project.api.entity';
+import { DbConnections } from '@marxan-api/ormconfig.connections';
 
 @Module({
-  imports: [CqrsModule, BlmModule],
+  imports: [
+    CqrsModule,
+    BlmModule,
+    TypeOrmModule.forFeature([Project]),
+    TypeOrmModule.forFeature([], DbConnections.geoprocessingDB),
```

Comment:
Shouldn't we load concrete entities?

Reply:
Here we just need the connection to do a raw SQL query